volunteer_planner/settings/sentry.py

```diff
@@ -0,0 +1,43 @@
+"""
+Imports and initializes sentry for usage within this Django instance.
+
+For successful initialization 'sentry-sdk' needs to be installed and some
+environment variables need to be present/set:
+- SENTRY_DSN: sentry data source name (https://docs.sentry.io/product/sentry-basics/dsn-explainer/)  # noqa: E501
+              mandatory, without no initialization will be successfully finished.
+- SENTRY_ENVIRON: string, used as 'environment' tag value
+                  default: 'production'
+- SENTRY_TRACE_RATE: float, (0..1), probability a transaction is performance monitored
+                     default: 0.0
+- SENTRY_SEND_PII: bool, (https://docs.sentry.io/platforms/python/configuration/options/#send-default-pii)  # noqa: 501
+                   default: False
+"""
+import logging
+import os
+
+from distutils.util import strtobool
+
+logger = logging.getLogger(__name__)
```

Comment:
Logger initialization could be done in except block

Reply:
It's two excepts. Is it really good style, to initialise logger at two locations?

Reply:
See other comments ;-)